src/qt/button.cpp

```diff
@@ -60,6 +60,17 @@ wxWindow *wxButton::SetDefault()
 
 }
 
+void wxButton::DoSetBitmap(const wxBitmap& bitmap, State WXUNUSED(which))
+{
+    m_bitmap = bitmap;
```

Comment:
I think this should only be done if `which == State_Normal` as this seems to be the only bitmap supported. Also, `DoGetBitmap()` below should only return `m_bitmap` in this case (and `wxNullBitmap` otherwise).

The reason for this is that it's going to be very surprising for a call to `SetBitmapDisabled()` to modify the normal bitmap, for  example.

Reply:
Ah ok, that's makes sense. I'll make those changes. Thanks.